src/test/java/org/gridsuite/filter/server/FilterEntityControllerTest.java

```diff
@@ -833,6 +838,158 @@ public void testIdentifierListFilter() throws Exception {
         checkIdentifierListFilterExportAndMetadata(substationFilterId, "[{\"id\":\"P1\",\"type\":\"SUBSTATION\"},{\"id\":\"P2\",\"type\":\"SUBSTATION\"}]\n", EquipmentType.SUBSTATION);
     }
 
+    @Test
+    public void testGetFiltersByIds() throws Exception {
+        UUID filterId3 = UUID.fromString("42b70a4d-e0c4-413a-8e3e-78e9027d300c");
+        UUID filterId4 = UUID.fromString("42b70a4d-e0c4-413a-8e3e-78e9027d300d");
+
+        LineFilter lineFilter = new LineFilter("NHV1_NHV2_1", null, "P1", "P2", new TreeSet<>(Set.of("FR")), new TreeSet<>(Set.of("FR")), new NumericalFilter(RangeType.RANGE, 360., 400.), new NumericalFilter(RangeType.RANGE, 356.25, 393.75));
+        CriteriaFilter lineCriteriaFilter = new CriteriaFilter(
+                filterId3,
+                new Date(),
+                lineFilter
+        );
+        insertFilter(filterId3, lineCriteriaFilter);
+        checkFormFilter(filterId3, lineCriteriaFilter);
+
+        LineFilter lineFilter2 = new LineFilter("NHV1_NHV2_1", null, "P1", "P2", new TreeSet<>(Set.of("FR")), new TreeSet<>(Set.of("FR")), new NumericalFilter(RangeType.RANGE, 360., 400.), new NumericalFilter(RangeType.RANGE, 356.25, 393.75));
+
+        CriteriaFilter lineCriteriaFilter2 = new CriteriaFilter(
+                filterId4,
+                new Date(),
+                lineFilter2
+        );
+
+        insertFilter(filterId4, lineCriteriaFilter2);
+        checkFormFilter(filterId4, lineCriteriaFilter2);
+
+        List<String> values = Arrays.asList(filterId3.toString(), filterId4.toString());
+        MultiValueMap<String, String> params = new LinkedMultiValueMap<>();
+        params.addAll("ids", values);
+
+        List<AbstractFilter> filterAttributes = objectMapper.readValue(
+                mvc.perform(get("/" + FilterApi.API_VERSION + "/filters/metadata").params(params)
+                                .contentType(APPLICATION_JSON))
+                        .andExpect(status().isOk())
+                        .andReturn().getResponse().getContentAsString(),
+                new TypeReference<>() {
+                });
+        Date dateModification = filterAttributes.get(0).getModificationDate();
+        Date dateModification2 = filterAttributes.get(1).getModificationDate();
+
+        var expected = "[ {\n" +
```

Comment:
Oh !
No hardcoded json string
Rather use DTO.builder and objectMapper.writeValueAsString